lib/edda-server/src/change_set_processor_task/materialized_view.rs

```diff
@@ -368,8 +395,32 @@ pub async fn build_mv_for_changes_in_change_set(
             change_set_id,
         })?;
     let from_index_checksum = index_frontend_object.checksum;
-    let (frontend_objects, patches) =
-        build_mv_inner(ctx, frigg, workspace_id, change_set_id, changes).await?;
+    let (
+        frontend_objects,
+        patches,
+        build_count,
+        build_total_elapsed,
+        build_max_elapsed,
+        build_slowest_mv_kind,
+    ) = build_mv_inner(
+        ctx,
+        frigg,
+        workspace_id,
+        change_set_id,
+        changes,
+    )
+    .await?;
+    span.record("si.edda.mv.count", build_count);
+    span.record(
+        "si.edda.mv.avg_build_elapsed_ms",
+        build_total_elapsed.as_millis() / build_count,
+    );
+    span.record(
+        "si.edda.mv.max_build_elapsed_ms",
+        build_max_elapsed.as_millis(),
+    );
+    span.record("si.edda.mv.slowest_kind", build_slowest_mv_kind);
```

Comment:
These `span.record()` calls would attempt to add values to this function's `materialized_view.build_mv_for_changes_in_change_set` Span, but these fields aren't set up in the `#[instrument()]` call, meaning that the values will be dropped.

Is it possible that you meant to add those 4 fields to this Span as well as to the `materialized_view.build_all_mv_for_change_set` Span?